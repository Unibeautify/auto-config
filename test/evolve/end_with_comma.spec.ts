import { testGenetic } from "../testGenetic";
import jsBeautify from "@unibeautify/beautifier-js-beautify";

testGenetic("end_with_comma", {
  beautifiers: [jsBeautify],
  language: "JavaScript",
  originalText: `var bar = {bar: "baz", qux: "quux"};`,
  desiredText: `var bar = {\n    bar: "baz",\n    qux: "quux",\n};`,
});
