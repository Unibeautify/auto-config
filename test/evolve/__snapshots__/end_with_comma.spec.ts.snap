// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`end_with_comma: beautifiedText 1`] = `
var bar = {
    bar: "baz",
    qux: "quux"
};
`;

exports[`end_with_comma: entity 1`] = `
Object {
  "options": Object {
    "beautifiers": Array [
      "JS-Beautify",
    ],
  },
}
`;

exports[`end_with_comma: fitness 1`] = `10000`;
